### DISCUSSION TITLE
HmIP-FSM - fehlende Messwerte - CCU vs. Home Assistant

### DISCUSSION THREAD

**Discussion by Michael-Hennemann** [SukramJ/aiohomematic#1504] [Q&A]
Hi, 
ich habe einen HmIP-FSM für mein kleines Balkonkraftwerk und festgestellt, dass es in meiner CCU einen Wert für den Tagesertrag gibt.
Außerdem noch Ertrag Vortag, letzte 7 Tage, etc. 

In Home Assistant kommt aber als Entity nur der Wert für den Gesamtertrag an. 

Gibt es eine Möglichkeit im HA an die anderen Messwerte heranzukommen? 

Viele Grüße und vielen Dank
Michael

**Comment by SukramJ** [comment]
Die findest du bei den Systemvariablen

**Reply by Michael-Hennemann** [reply]
Hi, vielen Dank für die schnelle Antwort!  Habs aber noch nicht ganz verstanden: Bei welchen Systemvariablen?  In der CCU/Raspimatic tauchen die Messwerte bei dem Gerät selbst auf nicht bei den Systemvariablen. Konnte auch keine hidden entity unterhalb "Homematic(IP) Local" Integration finden. Wo muss ich da im HA suchen? 
Cheers 
Michael

**Reply by SukramJ** [reply]
In HA tauchen die Werte als Systemvariablen auf, aber die Namen sind kryptisch, also viel Spass beim Suchen.
Systemvariaben findet Du mit klick auf die Integration unter Dienst, oder unter dem Gerät vom Typ CCU.
Die Systemvariaben müssen noch aktiviert werden.

**Comment by Baxxy13** [comment]
Nur "Energie-Zähler  - Zentrale / CCU / RM" ist eine systeminterne Systemvariable mit folgendem Namensschema:
`svEnergyCounter_24054_000B98A9AB90C2:5` 

`svEnergyCounter_<ID_DES_MESSWERTKANALS>_<SN_DES_GERAETES>:<KANALNUMMER_MESSWERTKANAL>`

Die anderen 4 "neuen" Zähler...
`Heute / Gestern / vergangene 7 Tage / vergangene 30 Tage`
...werden als Metadaten auf dem Messwertkanal gespeichert.
An die Metadaten kommt man auf der Zentrale selber mit HM-Script dran, Zugriff von "extern" ginge wohl mit Remote-Script, habe ich aber bisher nicht probiert.

Info's zum Thema: https://homematic-forum.de/forum/viewtopic.php?f=60&t=80952#p788442
